wwr_ui/randomizer_window.py

```diff
@@ -200,10 +228,30 @@ def randomize(self):
     self.ui.seed.setText(seed)
     self.update_settings()
     
+    important_options = [ # Options to track in spoiler log regardless of if they're default or not
+    "sword_mode",
+    "num_starting_triforce_shards",
```

Comment:
Why is `num_starting_triforce_shards` considered important but `randomize_entrances` isn't? Seems kinda arbitrary.

Reply:
Mainly because the default option for `randomize entrances` is "disabled" and I don't see this changing at any point in time, whereas `num_starting_triforce_shards` is just a number. Another would be because I asked lago about the starting shards and he said he thought it'd be better to keep that one in, but if that idea is ever changed, someone can feel free to change it since it's just a simple list.